Aws/Core.hs

```diff
@@ -449,22 +452,14 @@ data SignedQuery
         -- | Additional non-"amz" headers.
       , sqOtherHeaders :: !HTTP.RequestHeaders
         -- | Request body (used with 'Post' and 'Put').
-#if MIN_VERSION_http_conduit(2, 0, 0)
```

Comment:
If you're going to remove these checks, please make sure to also increase the minimum dependency version in aws.cabal.

Reply:
I think `aws.cabal` already says `http-conduit >= 2.1 && < 2.3`.
Is it not enough?